addons/a11y/package.json

```diff
@@ -20,7 +20,8 @@
     "directory": "addons/a11y"
   },
   "license": "MIT",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "types": "dist/index.d.ts",
```

Comment:
Let me check

Reply:
This is a problem... the types are still generated in the old location...

Should the typings be duplicated?
Should we compile to a /ts folder instead?

WDYT @gaetanmaisse ?

Reply:
> Should the typings be duplicated?
There is only one `types` property so we can not duplicate the types

> Should we compile to a /ts folder instead?
Not sure there are any benefits, except having trouble if we add a `cjs`/`esm` directory in our `src` but it will never happen...

Maybe everything is OK with the current setup, all CI checks are 🟢  which is a good thing. I can try this PR locally and see what happens 🤷🏻

Reply:
it works... but the output is messy. I think putting the typings into its own directory would look better?

Reply:
Agree, I think there will be some updates accordingly in the `downlevel-dts` part, maybe update the conf in the "build" and update paths in `package.json`s